### PR TITLE
feat(Radio): Add Radio Component DEV-136

### PR DESCRIPTION
**Pull request by welitonderesende**

### CODE REVIEW

**Review by VagnerNico — CHANGES_REQUESTED** [review]
You forgot to export Radio and RadioProps on the index.ts.

File: src/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  const radioGroup = useRadioGroup()
+
+  if (radioGroup) {
+    if (typeof onChange === "undefined") {
+      onChange = radioGroup.onChange
+    }
+    if (typeof name === "undefined") {
+      name = radioGroup.name
+    }
+  }
```
Comment (giubatt) [inline src/components/Radio/Radio.tsx:39]: Shouldn't it also use the `value` from the radioGroup context, when it's a controlled radio group?

**Comment by giubatt** [comment]
The example in storybook for the radiogroup doesn't seem to be working as expected
Peek 2021-05-24 14-41

File: src/components/Radio/Radio.tsx (hunk 1 above)
Comment (welitonderesende) [reply src/components/Radio/Radio.tsx]: done! ✅

**Review by giubatt — APPROVED** [review]

**Review by VagnerNico — APPROVED** [review]

**Review by VagnerNico — APPROVED** [review]

**Review by giubatt — APPROVED** [review]

**Pull request merged**